dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/subcommands/impl/All.java

```diff
@@ -35,10 +38,18 @@
 
     @Override
     public void execute(final Namespace ns, final DockerfileGitHubUtil dockerfileGitHubUtil)
-            throws IOException, InterruptedException {
+            throws IOException, InterruptedException, URISyntaxException {
         loadDockerfileGithubUtil(dockerfileGitHubUtil);
-        Set<Map.Entry<String, JsonElement>> imageToTagStore =
-                this.dockerfileGitHubUtil.getGitHubJsonStore(ns.get(Constants.STORE)).parseStoreToImagesMap(dockerfileGitHubUtil, ns.get(Constants.STORE));
+        String store = ns.get(Constants.STORE);
+        ImageStoreUtil imageStoreUtil = getImageStoreUtil();
+        ImageTagStore imageTagStore = imageStoreUtil.initializeImageTagStore(this.dockerfileGitHubUtil, store);
+        List<ImageTagStoreContent> imageNamesWithTag = imageTagStore.getStoreContent(dockerfileGitHubUtil, store);
+        AtomicInteger numberOfImagesToProcess = new AtomicInteger(imageNamesWithTag.size());
```

Comment:
I'm not sure why we need an AtomicInteger here?  This code does not appear multi-threaded an integer assignment in Java is already atomic.  See https://stackoverflow.com/questions/1006655/are-java-primitive-ints-atomic-by-design-or-by-accident .  If we don't need it just use `int`.